tests/src/com/unciv/testing/BasicTests.kt

```diff
@@ -63,12 +63,13 @@ class BasicTests {
 
     // If there's a unit that obsoletes with no upgrade then when it obsoletes
 // and we try to work on its upgrade, we'll get an exception - see techManager
+    // But...Scout obsoletes at Scientific Theory with no upgrade...?
```

Comment:
I had no idea this was hardcoded, we should probably add a unique for this, this is generally not supposed to happen